packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -165,6 +166,13 @@ export default {
          */
         legacyBanner () {
             return this.shouldShowLegacyBanner === null ? this.config.displayLegacy : this.shouldShowLegacyBanner;
+        },
+        /**
+         * Check if the reopen cookie link should use a grey background
+         * @returns {Boolean}
+         */
+        shouldUseGreyBackground () {
```

Comment:
Do you need this completed property? Seems fine to pass along the prop as you were before if you don't need any additional logic?

Reply:
Ah, is it so you can configure it in Storybook?

Reply:
Sorry, but I don't get the reason for this computed prop still

Reply:
Can't you just use one `shouldUseGreyBackground` prop without a computed one?

Reply:
@dandel10n Computed value removed